server/features/EmailInbox/EmailInbox_Incoming.ts

```diff
@@ -205,6 +221,7 @@ export async function onEmailReceived(email: ParsedMail, inbox: string, departme
 			},
 		});
 	}).catch((error) => {
-		Livechat.logger.log('Error receiving Email: %s', error.message);
+		console.log(error);
+		Livechat.logger.error('Error receiving Email: %s', error.message);
```

Comment:
same here with logging the error object. if you need to add a message to the log, you can add the `msg` key:

```
Livechat.logger.error({
  msg: 'Error receiving Email',
  err: error,
});
```